src/Components/Components/src/Reflection/ComponentProperties.cs

```diff
@@ -100,15 +101,15 @@ public static void SetProperties(in ParameterView parameters, object target)
                         // valid because cascading parameter names are not part of the public API. There's no
                         // way for the user of a component to know what the names of cascading parameters
                         // are.
-                        (writer.Cascading && !parameter.Cascading))
+                        (writer!.Cascading && !parameter.Cascading))
```

Comment:
It might make sense to reflow the code in a different way, like
```
if(!writers.WritersByName.TryGetValue(parameterName, out var writer)){
...
}
else
{
...
}
```